conf/nginx-alldomains.conf

```diff
@@ -26,9 +26,22 @@
 		index index.php;
 		alias /usr/local/lib/roundcubemail/;
 	}
-	location ~ /mail/config/.* {
-		# A ~-style location is needed to give this precedence over the next block.
-		return 403;
+	location ~ ^/mail/(config|temp|logs)/ {
+        	deny all;
+	}
+	location ~ ^/mail/(README|INSTALL|LICENSE|CHANGELOG|UPGRADING)$ {
+        	deny all;
+	}
+	location ~ ^/mail/(bin|SQL)/ {
+        	deny all;
+	}
+	location ~ ^/mail/(.+\.md)$ {
+        	deny all;
+	}
+	location ~ ^/mail/\. {
+        	deny all;
+        	access_log off;
```

Comment:
We can take it out (it's optional only), but it's using the same logic for other dot-files, too.

```
	# Disable viewing dotfiles (.htaccess, .svn, .git, etc.)
	# This block is placed at the end. Nginx's precedence rules means this block
	# takes precedence over all non-regex matches and only regex matches that
	# come after it (i.e. none of those, since this is the last one.) That means
	# we're blocking dotfiles in the static hosted sites but not the FastCGI-
	# handled locations for ownCloud (which serves user-uploaded files that might
	# have this pattern, see #414) or some of the other services.
	location ~ /\.(ht|svn|git|hg|bzr) {
		log_not_found off;
		access_log off;
		deny all;
	}
```